src/main/scala/com.snowplowanalytics.sauna/loggers/DDBLogger.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2015 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.sauna.loggers
+
+import com.snowplowanalytics.sauna.Sauna
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
+trait DDBLogger extends Logger {
+
+  /**
+    * Writes the message to DynamoDb table.
+    *
+    * @param uid Unique identifier for message.
+    * @param name Message header (for example, Optimizely list name).
+    * @param status HTTP code for operation result.
+    * @param description What happened.
+    * @param lastModified Last modification time, if exists. Else, when message was processed.
+    */
```

Comment:
I'm using Intellij IDEA too, it's annoyed me for some time. Then I found how to fix it: https://youtrack.jetbrains.com/issue/SCL-9609#comment=27-1245402
Probably it's very subjective, but my feeling is that in Scala community JavaDoc is more popular https://github.com/databricks/scala-style-guide#documentation-style